package.json

```diff
@@ -46,10 +47,10 @@
   ],
   "keywords": [],
   "dependencies": {
-    "babel-preset-es2015": "^6.5.0",
     "dom-scroll-into-view": "1.0.1",
-    "lodash": "^4.5.0",
-    "react": "^0.14.7",
-    "react-dom": "^0.14.7"
+    "lodash": "^4.5.0"
+  },
+  "peerDependencies": {
+    "react": "^0.14.7 || ^15.0.0-0"
```

Comment:
React.render was removed in version 15 - probably want to update the examples in this PR too.